index.html

```diff
@@ -10013,7 +10013,7 @@ <h3>Value</h3>
 				<dt id="valuetype_tristate">tristate</dt>
 				<dd>Value representing <code>true</code>, <code>false</code>, <code>mixed</code>, or <code>undefined</code> values. The default value for this value type is <code>undefined</code> unless otherwise specified.</dd>
 				<dt id="valuetype_true-false-undefined">true/false/undefined</dt>
-				<dd>Value representing <code>true</code>, <code>false</code>, or <code>undefined</code> (not applicable). The default value for this value type is <code>undefined</code> unless otherwise specified. For example, an element with <sref>aria-expanded</sref> set to <code>false</code> is not currently expanded; an element with <sref>aria-expanded</sref> set to <code>undefined</code> is not expandable.</dd>
+				<dd>Value representing <code>true</code>, <code>false</code>, or <code>undefined</code> (not applicable). Unless otherwise specified, the default value for this value type is <code>undefined</code>. Note that the value <code>undefined</code> conveys a state- or property-dependent behavior including but not limited to lack of support (e.g., <sref>aria-checked</sref>, <sref>aria-pressed</sref>), equivalence with the value <code>false</code> (e.g., <sref>aria-multiselectable</sref>), behavior determined by the user agent (e.g., <sref>aria-hidden</sref>) or non-applicability (e.g., <sref>aria-selected</sref>).
```

Comment:
I think it's actually even more complex than this :/. For example, `el.ariaChecked = undefined` is going to default to `false` for most roles, only excluding `option` and `treeitem`. I think it may be better to not reference specific attributes here. Even for something like `aria-pressed`, if HTML ever ends up creating a toggle button element, then `aria-pressed` might also conditionally default to false (and yeah, an element like that shouldn't use ARIA for that state, but if undefined it would still default to a boolean). It might be simpler to stick with only one relatively safe example (e.g. the `aria-expanded` one in the original text), then clarify that for specific behavior, authors should check the tables for that attribute?

I do also think I agree with Valerie that it'd be helpful to clarify that `undefined` here refers to the value, which is set by either leaving the attribute off entirely or explicitly setting e.g. `element.ariaExpanded = undefined`, and is not equivalent to the string `"undefined"` in e.g. `aria-expanded="undefined"`. What do you think?

Reply:
I think "undefined" needs to be yanked out of the `<code>` unless it explicitly means the string value "undefined".

As a _user_ of the spec, this is what I might expect (fake example) so I could make sense of it:
> Values include <code>yeah</code>, <code>nah</code>, <code>undefined</code>, or undefined (by setting no value).

Reply:
@aardrian would it help if this section had a link to the IDL reflection section just after it, in addition to text clarifying that `undefined` is not the string?

It does get weirder, in that while you can explicitly set e.g. `el.ariaThing = undefined`, getting it returns `null` 🤷🏻‍♀️

Reply:
Thank you @aardrian @smhigley. As @spectranaut noted, it's currently ambiguous if the value undefined is the string "undefined" or undefined as in not present. For example, see Note 2 under [aria-hidden](https://w3c.github.io/aria/#aria-hidden):

As of ARIA 1.3, [aria-hidden](https://w3c.github.io/aria/#aria-hidden)="false" is now synonymous with aria-hidden="undefined".

Reply:
@rahimabdi yeah, I believe that's a pretty straightforward mistake in the spec. I made a test page to output the browser-mapped values when ARIA attributes are set to the string `"undefined"`, not set at all, or set programmatically to `undefined`: https://jsfiddle.net/pobwvgs9/

I checked it in Chrome, Edge, Safari, and Firefox on macOS and Windows (Safari only on mac), and all of them had different results for the string `"undefined"`. The results for not-defined and programmatic `= undefined` values are what I'd expect & what matches the spec, while the string `"undefined"` are not.

My proposal would be to go through the spec and remove any instances accidentally using string `="undefined"`, and add the clarification of the string not being the same as the value.

As an aside, I do get an error for `="undefined"` in the W3C Validator, maybe they have some bugs in the attributes you tested?
<img width="927" alt="Screenshot of the HTML validator having been run on a button with aria-hidden=undefined and aria-pressed=foo. The validator shows one error for each attribute." src="https://github.com/w3c/aria/assets/3819570/7efcacf6-3c33-4d7d-9b3d-13258e008daa">